prairielib/lib/migrations.test.js

```diff
@@ -0,0 +1,226 @@
+// @ts-check
+/* eslint-env jest */
+
+const path = require('path');
+const tmp = require('tmp-promise');
+const fs = require('fs-extra');
+
+const {
+  readAndValidateMigrationsFromDirectory,
+  sortMigrationFiles,
+  getMigrationsToExecute,
+} = require('./migrations');
+
+async function withMigrationFiles(files, fn) {
```

Comment:
I really like these `with` helpers that you are using recently!

Reply:
<3 Still wishing JS had a `with` keyword like Python's that didn't uh, completely suck like the [*actual* JS `with` keyword](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/with) 😭